### PR TITLE
Starts work on the Vatborn Human subspecies

### PR DESCRIPTION
**Pull request by Anewbe**
TODO:
- [x] Finish getting Staff Approval
- [x] Need a Lorebook Blurb for Prometheans
- [x] Need a Lorebook Blurb for Vatborn

### CODE REVIEW

**Comment by Belsima** [comment]
i love you anewbe

**Comment by SilverTalismen** [comment]
What's the idea exactly behind the Vatborn in this universe?

**Comment by Anewbe** [comment]
- Basic concept is "Specialized genetic template that gets quick-grown to adulthood in a big tube, has minor biological tweaks that the creators thought would be beneficial, or that came about as minor side-effects."
- Niche they fill in society is "Still technically a person, but clinging desperately to the line between accepted genetic modification and Five Points violations."

**Comment by SilverTalismen** [comment]
So do they have rights or are they viewed as expendable, and are others able to have them created specifically?

**Comment by Anewbe** [comment]
> Niche they fill in society is "Still technically a person, but clinging desperately to the line between accepted genetic modification and Five Points violations."

**Comment by SilverTalismen** [comment]
Ahh okay. Sorry I don't really understand what that last part means, five points violation?

**Comment by Anewbe** [comment]
Lore thing, basically the government said "These Five Things will ruin humanity, don't do them." Certain types of Genetic Modification are three of those five.

https://doku.ss13polaris.com/doku.php?id=lore:five-points&s[]=five&s[]=points

**Comment by SilverTalismen** [comment]
Aah right, okay thank you for the clarification, will the Vatborn look any different? I imagine they will have a limit on command positions and stuff.

**Comment by Anewbe** [comment]
They're like... Nearly white, right now.
And no, they're just humans. Some people aren't a fan, but NanoTrasen Corporate doesn't particularly care.

**Comment by SilverTalismen** [comment]
Could always find someone to get some special markings for them to be 'branded' with so people know.

**Comment by Anewbe** [comment]
... Why?

**Comment by SilverTalismen** [comment]
I mean, I assume there would be a marking or tattoo of some sort so they can tell a Vatborn from normal humans if they don't have access to like, records or something for whatever reason?

**Comment by Anewbe** [comment]
There's not a reason to do so. It's like an IVF baby.

**Comment by SilverTalismen** [comment]
I suppose, It was mostly a idea to add some flavor not a thing that needed to happen ^^ Anyway I look forward to seeing Vatborn around and possibly even applying for one if thats such a thing.

**Comment by Anewbe** [comment]
Nope, gonna be something anyone can take.

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -701,7 +701,7 @@
 	elvis
 		name = "Elvis Sideburns"
 		icon_state = "facial_elvis"
-		species_allowed = list("Human","Unathi")
+		species_allowed = list("Human","Promethan","Vatborn","Unathi")
```
Comment (elgeonmb) [inline code/modules/mob/new_player/sprite_accessories.dm]: Should be "Promethean"

Comment (Anewbe) [reply code/modules/mob/new_player/sprite_accessories.dm]: Good catch.

Comment (elgeonmb) [reply code/modules/mob/new_player/sprite_accessories.dm]: There's a lot of 'em that are typoed.

Comment (Anewbe) [reply code/modules/mob/new_player/sprite_accessories.dm]: I'm bad at words.

**Comment by Belsima** [comment]
they are not 'just humans'... they are mistakes

**Comment by Anewbe** [comment]
That's the spirit.

**Comment by Mechoid** [comment]
Looks good code-wise. Is there any more discussion needed? Tested, etcetera?

**Comment by Anewbe** [comment]
It runs, staff-side discussion is complete.

**Comment by elgeonmb** [comment]
👍 👌 can't wait for very little of consequence to change

**Pull request merged**